MaterialLib/MPL/mpPhase.cpp

```diff
@@ -0,0 +1,134 @@
+/**
+ * \author Norbert Grunwald
+ * \date   Sep 7, 2017
+ * \brief
+ *
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "mpPhase.h"
```

Comment:
The same: the class name is different from the file name.

Reply:
There are other files have the same issue: the class name in the file  is different from the file name.